controller.rst

```diff
@@ -116,19 +116,30 @@ For more information on routing, see :doc:`/routing`.
 .. index::
    single: Controller; Base controller class
 
-The Base Controller Class & Services
-------------------------------------
+.. _anchor-name:
+   :ref:`The Base Controller Classes & Services <the-base-controller-class-services>`
```

Comment:
Yes that was [a request from @xabbuh](https://github.com/symfony/symfony-docs/pull/7657#discussion_r109855847).
Thanks for the example, I never did that before and obviously didn't do it well by randomly trying :)